sale_cancel_reason/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013 Guewen Baconnier, Camptocamp SA
+# Copyright 2019 Victor M.M. Torres, Tecnativa SL
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Sale Cancel Reason',
+    'version': '12.0.1.2.0',
```

Comment:
As you ported this from 12.0 to 11.0, you need to change the module version too
```suggestion
    'version': '11.0.1.0.0',
```
This is why your build is currently failing.

Reply:
@mmequignon Ah yes, of course. Thanks.